docs/FAQs.md

```diff
@@ -3,11 +3,19 @@ Frequently Asked Questions
 
 ### Why does Test Pilot require a Firefox Account?
 
-Test Pilot was launched with the requirement that particpants be logged into a Firefox Account.
+Test Pilot was launched with the requirement that participants be logged into a Firefox Account.
```

Comment:
Well, @ckprice sits close enough that he can walk over and punch me, so I'm only a tough guy on The Internet when I'm confident that I'm safe.

PS: Please don't beat me up in London.